index.html

```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Document </title>
     <link rel="stylesheet" href="style.css"> 
 </head>
 <body>
-    <h1>Hello  Microverse </h1>
+    <h1>Hello Microverse </h1>
 
     
```

Comment:
## 3.
![PR-TITLE (copy)](https://user-images.githubusercontent.com/93039359/184926275-ca39df01-92cb-4067-b5d7-af3bcf604e52.png)


- [ ] Your branch name is not descriptive enough. You are using **dev**, you can use  **feature/setup-hello-microverse** or **feature/initialize-hello-microverse** :+1: